src/deprecated/src/integrations/third-party/the-events-calendar.php

```diff
@@ -5,17 +5,21 @@
 use Yoast\WP\SEO\Conditionals\Front_End_Conditional;
 use Yoast\WP\SEO\Conditionals\Open_Graph_Conditional;
 use Yoast\WP\SEO\Conditionals\The_Events_Calendar_Conditional;
-use Yoast\WP\SEO\Generators\Schema\Third_Party\Events_Calendar_Schema;
 use Yoast\WP\SEO\Integrations\Integration_Interface;
 
 /**
  * Class The_Events_Calendar
+ *
+ * @deprecated 19.12
  */
 class The_Events_Calendar implements Integration_Interface {
 
 	/**
 	 * Returns the conditionals based in which this loadable should be active.
 	 *
+	 * @deprecated 19.12
+	 * @codeCoverageIgnore
+	 *
 	 * @return array
 	 */
 	public static function get_conditionals() {
```

Comment:
Aside from the `@deprecated 19.12` line in the function docblock, it also needs a `\_deprecated_function( __METHOD__, 'WPSEO 19.12' );` inside the `get_conditionals()` method too